impl/maven-impl/src/test/java/org/apache/maven/impl/DefaultPluginXmlFactoryTest.java

```diff
@@ -160,18 +161,18 @@ void toXmlStringGeneratesValidXml() {
                 .artifactId("sample-plugin")
                 .version("1.0.0")
                 .build());
-        assertThat(xml).contains("<name>" + NAME + "</name>");
-        assertThat(xml).contains("<groupId>org.example</groupId>");
         assertThat(xml).contains("<artifactId>sample-plugin</artifactId>");
+        assertThat(xml).contains("<groupId>org.example</groupId>");
```

Comment:
why reorder?

Reply:
give natural order but group is first, yes.